compute/gpu/concepts.mdx

```diff
@@ -16,22 +16,102 @@ categories:
 
 <Concept opened>
 
-## Docker
+## AI-Enabled applications
 
-[Docker](https://www.docker.com/) is a platform as a service (PaaS) tool, that uses OS-level virtualization to deliver software in packages called containers. Scaleway provides a number of pre-built Docker images, which allow you to [run a Docker container on your GPU Instance](/compute/gpu/how-to/use-gpu-with-docker/) and access a pre-installed Pipenv environment ideal for your AI projects.
+The proliferation of AI-Enabled applications and innovative use cases is poised to bring transformative changes to various domains, including business, social interactions, and human-machine interfaces.
+These applications will feature conversational and multimodal user interfaces (UIs) that will revolutionize interactions within smart spaces, smart robots, and autonomous vehicles.
```

Comment:
I'd put this bit first, since it actually defines what they are, and then "The proliferation of..." after